### PR TITLE
recycle color

### PR DESCRIPTION
**Pull request by fandomfilms**

### CODE REVIEW

File: src/scss/misc/_colors.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```
Comment (bvoelker) [inline src/scss/misc/_colors.scss]: What happened to figuring out what we want our colors to be?

Comment (Luiz-N) [inline src/scss/misc/_colors.scss]: This is a short term solution to allow recycling to be much clearer on the frontend. Our next set of changes to the icons/map will be much more comprehensive with different shapes and colors.

Comment (bvoelker) [inline src/scss/misc/_colors.scss]: The problem i have is this wont be a short term. What will end up happening is well put this up and use this and wont take the time to go back and actually update the colors to what they should be. Case in point Coy has had the task of updating our colors for a very long time. and the answer always is "we dont have time". We cant keep taking shortcuts for everything we do it is adding too much debt and we are getting crushed by it. Have coy finish that up and lets roll out what the colors should be.

Comment (Luiz-N) [inline src/scss/misc/_colors.scss]: It isn't just the colors that will change though.. the shapes are also going to change by asset but changing shapes with our current map solution isn't easy as we learned yesterday. The comprehensive changes that we are going to make will come when we eventually switch to mapbox-gl but for now this is a small change for just a couple dozen wells in one state.

Comment (bvoelker) [inline src/scss/misc/_colors.scss]: Just talking about colors right now. We dont have to change shapes in order to pick colors. He is so close to a good color palette to use.

Comment (Luiz-N) [inline src/scss/misc/_colors.scss]: We don't know what the final color/shape solution will look like until we test the different options with our users and that won't be for a little while. We might use shapes to differentiate assets and let colors signify something else. Coy has decided red will be our recycle color until we hear differently from our users when that time comes.

**Pull request merged**